src/rules.js

```diff
@@ -200,13 +205,7 @@ module.exports = [
         'allowParamToValue': false,
         'styles': {
             'aspect-ratio': '$0'
-        },
-        'arguments': [{
-            'ini': 'initial',
-            'r': 'revert',
```

Comment:
So this is a breaking change?  Could we just use `r` for `revert` in the new globals, or are there reasons that wouldn't be safe?

Reply:
this is a new rule we recently added so the risk is small

Reply:
is this not a breaking change?

Reply:
It was a brand new rule that we just added so I was confident that changing it would not affect anyone.